nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```
@@ -236,10 +265,42 @@ in {
       '';
     };
 
+    secureBoot = mkOption {
+      default = {};
+      type = types.submodule ({ config, ... }: {
+        options = {
+          enable = mkEnableOption "the Lanzaboote's SecureBoot implementation";
```

This would complete to ``Whether to enable the Lanzaboote's SecureBoot implementation.`` I am not a native speaker but that sounds odd.

```suggestion
          enable = mkEnableOption "Lanzaboote's SecureBoot implementation";
```